dist/angular-validation.js

```diff
@@ -346,6 +347,10 @@
                     var messageElem,
                         messageToShow = validMessage || $validationProvider.getDefaultMsg(validation).error;
 
+                    if(!ctrl.$error[ctrl.$name] && ctrl.$dirty && ctrl.$invalid){
```

Comment:
@RichardSilveira I doubt he will accept this PR, your changes need to be moved to the src/* files and not dist. src/* are built via grunt and the produced scripts end up in dist.